app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -201,13 +200,16 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
   return (
     <>
       <PageHeading>{t('in_person_proofing.headings.po_search.location')}</PageHeading>
-      <p>{t('in_person_proofing.body.location.po_search.po_search_about')}</p>
-      {arcgisSearchEnabled && (
-        <AddressSearch onAddressFound={handleFoundAddress} registerField={registerField} />
-      )}
-      <p>{t('in_person_proofing.body.location.location_step_about')}</p>
-      {locationsContent}
-      <BackButton onClick={toPreviousStep} />
+      <div aria-live="polite">
+        {arcgisSearchEnabled && (
+          <AddressSearch onAddressFound={handleFoundAddress} registerField={registerField} />
+        )}
+      </div>
+      <div role="status">
```

Comment:
This would make the region a live region, meaning I'd expect a screen reader to read the entire contents of this region at the first opportunity. Is that expected? Since there's quite a lot of location content, would we want to let the user navigate this themselves, since it's not really a "status" text?

Reply:
Yes, that was the screen reader behavior that I expected.  I had interpreted these portions of the ticket  
> I want the screen reader to read out the correct order of the search page experience 

and 

> Flow of the reader should follow the hierarchy of the page

to mean that the screen reader should read the about text and then the locations text.  

You’re right that it is a lot of content, and I don’t personally use a screen reader, so I could be missing something.  I will check in with product/design about your concerns and then get back to you.

Reply:
We already verbally talked about this in a meeting yesterday.  However, for posterity’s sake, I wanted to note that I checked in with design/product yesterday and there is a [new ticket](https://cm-jira.usa.gov/browse/LG-8398) for the updated screen reader implementation.